packages/react-ui-core/src/Collapsible/Collapsible.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '../Button'
+
+const noop = () => {}
+
+export default class Collapsible extends Component {
+
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+    this.state = {
+      display: this.props.visible,
+    }
+  }
+
+  handleClick() {
+    this.props.handleClick()
+    this.setState({
+      display: !this.state.display,
+    })
+  }
+
+  render() {
+    const {
+      id,
+      theme,
+      showableItems,
+      nonshowableItems,
+      hiddenText,
+      visibleText,
+    } = this.props
+
+    return (
+      <div id={id} className={theme.collapsible}>
+        {showableItems}
+        <div className={this.state.display ? theme.show : theme.hide}>
+          {nonshowableItems}
+        </div>
+        <div className={theme.alignBottom}>
+          <Button className={theme.strong} onClick={this.handleClick}>
+            {this.state.display ? visibleText : hiddenText}
+          </Button>
+        </div>
+      </div>
+    )
+  }
+}
+
+Collapsible.defaultProps = {
```

Comment:
can you move `propTypes` and `defaultProps` up, into the object?  [See this for example](https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Button/Button.js#L11). The style you have here is fine if you're writing a functional component, but we prefer `static propTypes` and `static defaultProps` when extending `Component` or `PureComponent`.